+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Thu Sep 21 17:22:48 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL aide
+ pkgs=aide
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs aide --node 1 --tempfile ''
Thu Sep 21 17:23:02 2023 - INFO  - pkgs:(aide) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ sed -i '$a/home DATAONLY' /etc/aide.conf
+ aide --init
Start timestamp: 2023-09-21 17:23:02 +0000 (AIDE 0.18.5)
AIDE successfully initialized database.
New AIDE database written to /var/lib/aide/aide.db.new.gz

Number of entries:	178798

---------------------------------------------------
The attributes of the (uncompressed) database(s):
---------------------------------------------------

/var/lib/aide/aide.db.new.gz
 MD5       : mkVidNCNyNBidlEeGd6eJA==
 SHA1      : tv9K1INUeoJKjqIZ/SBdEG1qnZM=
 SHA256    : PskvF5ymc1tDu4nywvKXeRxPEvbDwSii
             3K43GOgkG50=
 SHA512    : H9HFwvRTeIormA//5SzPQtvnZG1TnLFA
             7kNVNtCuiCA1vcwNBknlta3E2X2FIWZ2
             dE+UH7cZINYx0pS4GlGdSQ==
 RMD160    : zXQvBsw+o6lorbIaliuGGZt+GMo=
 TIGER     : vtMX5iECO4EP0W79+lW6jPhZqVeMqm0b
 CRC32     : APOaQA==
 WHIRLPOOL : lZ5fQsVf3J+Mp4XGfy/SHRKXaRbuphX1
             PmaLir/sBp+Re+ps3Gz9jrpWQXGu+N5n
             yl6ADcUWG6xKoyBAazMvFA==
 GOST      : +VcIXEKyPdEx2AwZDGYqz7LF5rS9RTaL
             kORra4hwMaI=
 STRIBOG256: ZnApf/d9f6juk7CF0crJc4fMUxddrCFh
             3b+HSdpy2Pk=
 STRIBOG512: X2hp1zutjBPo5wfQVEjGaPUMAz1ko0dd
             9qRTjwA9QXjr8e+gzSZDI2eKVk/NMXxQ
             2oORfM4eZ2VNWkZLIzpo6w==
 SM3       : rXiUlTNLblZdWQEOwSZPxkn2iHkM43NA
             cuy/s9F5apw=


End timestamp: 2023-09-21 17:39:39 +0000 (run time: 16m 37s)
+ mv /var/lib/aide/aide.db.new.gz /var/lib/aide/aide.db.gz
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Thu Sep 21 17:39:40 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Sep 21 17:39:41 2023 - INFO  - Start to run test.
+ aide --config=/etc/aide.conf
+ grep -E 'Total number of entries|Added entries|Changed entries|Removed entries'
Thu Sep 21 17:52:43 2023 - ERROR - Timeout : Command 'bash -x oe_test_aide_config_cfgfile.sh' timed out after 1799.9995836999988 seconds
  Total number of entries:	178798
  Added entries:		0
  Removed entries:		0
  Changed entries:		1
Changed entries:
++ post_test
++ LOG_INFO 'Start to restore the test environment.'
++ message='Start to restore the test environment.'
++ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Sep 21 18:04:45 2023 - INFO  - Start to restore the test environment.
++ DNF_REMOVE
++ node=1
++ pkg_list=
++ mode=0
++ [[ -z '' ]]
++ [[ -z '' ]]
++ LOG_WARN 'no thing to do.'
++ message='no thing to do.'
++ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Thu Sep 21 18:04:45 2023 - WARN  - no thing to do.
++ return 0
++ rm -rf /var/lib/aide
++ LOG_INFO 'End to restore the test environment.'
++ message='End to restore the test environment.'
++ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Sep 21 18:04:46 2023 - INFO  - End to restore the test environment.
+ CHECK_RESULT 0 0 0 'Get aide configuration items: failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Get aide configuration items: failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Sep 21 18:04:47 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Thu Sep 21 18:04:47 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Sep 21 18:04:48 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Thu Sep 21 18:04:49 2023 - WARN  - no thing to do.
+ return 0
+ rm -rf /var/lib/aide
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Sep 21 18:04:50 2023 - INFO  - End to restore the test environment.
